lib/3scale_toolbox/commands/service_command/service_create.rb

```diff
@@ -0,0 +1,33 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceCreateSubcommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+
+        def self.command
+          Cri::Command.define do
+            name        'create'
+            usage       'create'
```

Comment:
something like this?
`create [opts] <source> <service-name>`

Reply:
:+1: